### ISSUE TITLE
BlenderBim - When using custom Css drawing create drawing renders a screenshot

### ISSUE THREAD

**Issue by Plae-Blue-Dot** [IfcOpenShell/IfcOpenShell#3840]
Using this version of Blender and BlenderBim:
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/78321676/edb5bf03-2e88-4a82-9cd6-40194657db70)

When changing the plan css references to custom ones it just takes renders a kind of screenshot instead of printing

default works fine:
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/78321676/25771d5b-a3af-4128-92f3-674cac607558)

Custom css file gives a kind of screenshot for the projection:
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/78321676/b7bf96b3-6008-4992-ba83-75e34086fd7d)

Before today never seen this issue, even on this file

here are the css/svg files
[markers_moonlit.svg.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/12790968/markers_moonlit.svg.txt)
[Moonlit_style.css.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/12790969/Moonlit_style.css.txt)
[patterns_moonlit.svg.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/12790970/patterns_moonlit.svg.txt)
[symbols_moonlit.svg.txt](https://github.com/IfcOpenShell/IfcOpenShell/files/12790971/symbols_moonlit.svg.txt)

The ifc file is private work I can share it to email if needed

**Comment by Moult** [comment]
Yes please, please share the IFC to dion@thinkmoult.com and let me know how to recreate it :) Sorry for the delay catching up on bugs!

**Comment by Plae-Blue-Dot** [comment]
No worries on the delay just sent the file! thanks Dion!

**Comment by sboddy** [comment]
@Plae-Blue-Dot Still an issue? Or would you like to keep it open?

**Comment by Plae-Blue-Dot** [comment]
This is sorted! Closing the issue dankie
